ContentFiles/ElementPackages/StandardLibrary/StandardPaths.ele

```diff
@@ -0,0 +1,91 @@
+namespace StandardPaths
+{
+    # Standard Paths are pre-defined Paths. They all have the domain (0, 1)
+    #
+    # If the path length is possible to calculate analytically, a
+    # separate function is defined to provide it.
+
+    circle(radius:Num):Path
+    {
+        # Return a clockwise circle in the x-y plane, beginning at
+        # x=0, y=radius
+        circleFunction(u:Num):Vector3
+        {
+            x = radius.mul(Num.sin(u.mul(Num.tau)))
+            y = radius.mul(Num.cos(u.mul(Num.tau)))
+            return = Vector3(x, y, 0)
+        }
+        return = Path(circleFunction, Bounds(0, 1))
+    }
+    circleLength(radius:Num):Num = Num.tau.mul(radius)
```

Comment:
Bear in mind this breaks if non-uniform scaling is applied. I think you mentioned that there was some information in the documentation somewhere to indicate that these functions are invalid if transformations are applied?

Reply:
Actually, that applies to all, doesn't it? For example the radius input parameter will not be transformed by any outer transform, hence this function returns the wrong length as so as _any_ transformation is applied.

Reply:
Yep, if you apply any transformation to the `Path`, then these lengths may change.

I could add this reminder to the top of the namespace:

```
    # If the path length is possible to calculate analytically, a separate
    # function is defined to provide it. Note that applying transforms
    # to the paths may change the path length.
```

I also mention in the `Path.pingPong` and `Path.repeat` docs that using these functions may change the path length. I hope this is sufficient?